test/e2e/upgradetest/resize_test.go

```diff
@@ -29,6 +29,9 @@ func TestResize(t *testing.T) {
 		if err != nil {
 			t.Fatal(err)
 		}
+		if err = e2eutil.WaitUntilDeploymentDeleted(testF.KubeCli, 3*time.Second, 30*time.Second, "etcd-operator", testF.KubeNS); err != nil {
```

Comment:
DeleteDeployment() doesn't set grace period, which is 30s by default. We can reduce that to 1s? What about 0s, what does it mean for deployment?

Do you know if deployment deleted means pod is also deleted?